slo_generator/backends/prometheus.py

```diff
@@ -97,7 +98,12 @@ def good_bad_ratio(self, timestamp, window, slo_config):
         return (good_count, bad_count)
 
     # pylint: disable=unused-argument
-    def distribution_cut(self, timestamp, window, slo_config):
+    def distribution_cut(
+        self,
+        timestamp: int,
+        window: int,
+        slo_config: dict
+    ) -> Tuple[float, float]:
```

Comment:
Yeah right, but it's returning float 🤷‍♂️

Still waiting for Olivier's response 👀

Reply:
Mmmh, need to look over the Prometheus response in that case - in some bakends good / bad are `float` because it's an average over a time window, I'm afraid turning this to `int` will decrease the overall precision of the resulting SLI...

Reply:
Let's keep `float` here. I will open an issue to confirm the actual type later on.